src/bgw/job.c

```diff
@@ -160,15 +160,15 @@ bgw_job_scan_job_id(int32 bgw_job_id, tuple_found_func tuple_found, tuple_filter
 					 scankey, 1, tuple_found, tuple_filter, data, mctx, lockmode);
 }
 
-static bool
+static ScanTupleResult
 bgw_job_tuple_found(TupleInfo *ti, void *const data)
 {
 	BgwJob	  **job_pp = data;
 
 	*job_pp = bgw_job_from_tuple(ti->tuple, sizeof(BgwJob), ti->mctx);
 
-	/* return true since used with scan one */
-	return true;
+	/* return SCAN_CONTINUE since used with scan one */
```

Comment:
I don't think these comments are entirely accurate. We return `SCAN_CONTINUE` because it's used as a `tuple_found` func handle, not because it's used in `scan_one`? Maybe a comment that's more `return SCAN_CONTINUE since used as a tuple_found function`. Or, just remove the comment since the naming of the function makes clear it's being used as as `tuple_found`.

Reply:
Yeah, I wasn't actually sure what the comment meant to begin with so I left it, but updated it. I think removing it makes sense.

Reply:
Although, I wondering if we shouldn't return `SCAN_DONE` here for clarity.